examples/pytest/test_demo.py

```diff
@@ -83,3 +84,18 @@ def test_nested_objects(snapshot):
     snapshot.assert_match(tuple_, 'tuple')
     snapshot.assert_match(set_, 'set')
     snapshot.assert_match(frozenset_, 'frozenset')
+
+
+def test_snapshot_can_ignore_keys(snapshot):
+    snapshot.assert_match(
+        {
+            "id": uuid.uuid4(),
```

Comment:
Given that single quotes are used for strings in other places, I'd stick to that.
See https://pep8.org/#string-quotes 
```
In Python, single-quoted strings and double-quoted strings are the same. This PEP does not make a recommendation for this. Pick a rule and stick to it.
```

Reply:
Ah I've gotten so used to black formatting, so didn't notice. Agree this should be consistent, will change.

Reply:
Hm looking closer at the code I can see both double quotes and single quotes are used at this point in the project.

This is a problem in itself and should be addressed in a separate PR imo. Preferably with black formatting if you ask me since I think formatting beats linting. And at the same time linting this should be added to enforce it. (`black --check` if black i chosen)

I don't really care if it's single or double quote just consistent :)